renovate.json

```diff
@@ -31,5 +30,18 @@
       "matchBaseBranches": ["main"],
       "enabled": true
     }
-  ]
+  ],
+  "lockFileMaintenance": {
+    "enabled": true,
+    "assignees": ["@joshuadkitenge", "@joelvdavies"],
+    "prBodyNotes": ["- Please ensure that `yarn dev` works correctly."]
+  },
+  "vulnerabilityAlerts": {
+    "enabled": true,
+    "labels": ["security"],
+    "assignees": ["@joshuadkitenge", "@joelvdavies"],
+    "prBodyNotes": [
+      "- Please ensure that `yarn dev` works correctly.\n- Once this PR has been merged, please follow the release process instructions here: https://github.com/ral-facilities/inventory-management-system/wiki/Release-Process.\n\nAdditionally, please verify if this PR can be grouped with others before being released."
```

Comment:
I think the \n needs to be \<br>

Reply:
I thought it wouldn't be possible to tell until merged but then I tried writing it and <br> created a new line unless I escaped it.